src/main/java/teammates/ui/webapi/GetFeedbackQuestionsAction.java

```diff
@@ -102,4 +150,12 @@ public JsonResult execute() {
         return new JsonResult(response);
     }
 
+    private List<FeedbackQuestionAttributes> convertToFeedbackQuestionAttributes(List<FeedbackQuestion> questions) {
```

Comment:
should not be using attributes class anymore.

Reply:
Alright, got it, I see that @hhdqirui's PR has done that, will pull from his branch once it's merged